.goreleaser.yaml

```diff
@@ -0,0 +1,129 @@
+# run a local "snapshot" build without publishing:
+#    TRAVIS_BUILD_NUMBER=1 goreleaser --snapshot --rm-dist --debug
+#
+project_name: chezmoi
+
+builds:
+- binary: chezmoi
+  env:
+    - CGO_ENABLED=0
+  ldflags:
+     - -s -w
+  goos:
+    - linux
+    - darwin
+    # TODO: uncomment when compatible with windows. currently fails on 'undefined: syscall.Stat_t'
+    # - windows
+    - freebsd
+    - openbsd
+    # - dragonfly
+    # - netbsd
+  goarch:
+    - 386
+    - amd64
+    - arm
+    - arm64
+  goarm:
+    - ''
+  ignore:
+    - goos: darwin
+      goarch: 386
+    - goos: windows
+      goarch: 386
+
+## generate .tar.gz and .zip archives
+archive:
+  replacements:
+    386: i386
+  format_overrides:
+  - goos: windows
+    format: zip
+
+checksum:
+  name_template: 'checksums.txt'
+
+snapshot:
+  name_template: "{{ .Tag }}-next"
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+    - '^docs:'
+    - '^test:'
+    - Merge pull request
+    - Merge branch
+
+## generate RPM and DEB packages
+nfpm:
+  vendor: "TODO"
+  homepage: "https://github.com/twpayne/chezmoi"
+  maintainer: Tom Payne <TODO@TODO.tld>
```

Comment:
You probably want to modify this. I'm not sure if you need an email address here. My guess is that the field is generic and name + email is just a convention.